release/src/version-info.unit.spec.ts

```diff
@@ -39,29 +39,29 @@ describe("version-info", () => {
       older: [],
     } as VersionInfoFile;
 
-    it("should add new latest version to version info json", () => {
+    it("should add new version to version info json", () => {
       const generatedJson = generateVersionInfoJson({
         milestoneIssues: issues,
         version: "v0.3.0",
         existingVersionInfo: oldJson,
       });
 
-      expect(generatedJson.latest).toEqual({
+      expect(generatedJson.older).toEqual([{
```

Comment:
Given the way that versions are added to `older` before being marked as latest, the naming feels funny. How about using the key `other` instead?

Reply:
I agree, `other` would be a much better name, but I think we need to maintain the current name for the sake of backward compatibility with existing instances.